public/_headers

```diff
@@ -1,3 +1,6 @@
 /assets/*
-  cache-control: max-age=86400
+  cache-control: max-age=604800
+  cache-control: public
+/_astro/*
+  cache-control: max-age=604800
```

Comment:
Yep, that'll work here too!  If I remember right CDNs will treat `immutable` the same as `public` but it will tell browsers to not even bother checking if the file has changed as long as the cache hasn't expired yet